pkg/common/common.go

```diff
@@ -29,7 +29,8 @@ const (
 	DiskImageName = "disk.img"
 	// ImporterWritePath provides a constant for the cmd/cdi-importer/importer.go executable
 	ImporterWritePath = ImporterVolumePath + "/" + DiskImageName
-
+	// ImporterWriteBlockPath provides a constant for the path where the PV is mounted.
+	ImporterWriteBlockPath = "/dev/loop0"
```

Comment:
We will need to resolve the path from the PV

Reply:
This is the path inside the pod. Can be a const